src/xdsl/xdsl_opt_main.py

```diff
@@ -76,7 +77,6 @@ def run(self):
     def register_all_arguments(self, arg_parser: argparse.ArgumentParser):
         """
         Registers all the command line arguments that are used by this tool.
-
```

Comment:
There are still missing linebreaks in this file :)

Reply:
I've tried to fix them manually since I was having issues w yapf. Feel free to alert me if its missing in other places.